### ISSUE TITLE
Comments deleted on registration success

### ISSUE THREAD

**Issue by aliblackwell** [coralproject/talk#3418]
**Expected behavior:**

User reads article and fills out comment box. Clicks "Sign in and join the conversation" button to submit their comment. Registers an account using email and password. Clicks submit and their comment is submitted.

**Actual behavior:**

When the user clicks submit, the Coral Talk iframe reloads, taking their comment with it. 

**Related Issues:** 

None.

**Versions:**

- NodeJS: 12.19.0
- NPM: 6.14.8
- MongoDB: 4.2 
- Redis: 3.2
- Browser: Firefox Developer Edition
- OS: Windows 10

You are welcome to verify this behaviour on our development server, the homepage of which pulls in Coral Talk: https://fake-od-site.staging-caprover.opendemocracy.net/

Any guidance on how to achieve the expected behaviour would be greatly appreciated. Personally, I always copy the contents of a comment box / contact form to my clipboard before registering, but I've been stung and want to make sure none of our users are.

I imagine lots of new users will write their comment before registering, so this bug has the potential to delete a great number of comments.

Thank you.

**Comment by DenisDoroshchuk** [comment]
Hey @wyattjoh
I have a same problem.
Until this issue is resolved, is there any workaround?

For example:
I know what comment was written. I can to get it from event "commentCount" after reload iframe.
But I don't know how I can to add this text to comment area http://joxi.ru/bmoyERVTyLOVxA

Thanks
Denis

**Comment by frisi** [comment]
fwiw @DenisDoroshchuk: we worked around this issue by hiding the comment input for not-signed in users.

```
/* hide comment box for not logged in users. otherwise comment text can get lost with click on "signup" button
see bugreport https://github.com/coralproject/talk/issues/3425
*/
.coral-unauthenticated .coral-rte {
  display: none;
}
.coral-unauthenticated div[class^='PostCommentFormFake'] {
  display: none;
}
```

**Comment by DenisDoroshchuk** [comment]
Hey @frisi, big thanks, it's working!

**Comment by wyattjoh** [comment]
We should have this issue addressed in the next release!

**Comment by immber** [comment]
This was resolved in v6.9.0 with this PR https://github.com/coralproject/talk/pull/3431
